### PR TITLE
[master] added Lesson07

### PR DESCRIPTION
**Pull request by panser81**

### CODE REVIEW

File: Lesson07/src/com/gmail/panser81/lesson07/part1/Circle.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.gmail.panser81.lesson07.part1;
+
+public class Circle extends Figure {
+    private double radius;
+    private static final double PI = 3.1415;
```
Comment (aperavoznikau-idt) [inline Lesson07/src/com/gmail/panser81/lesson07/part1/Circle.java:5]: Better to use Math.PI

File: Lesson07/src/com/gmail/panser81/lesson07/part1/Figure.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        return this.perimeter;
+    }
+
+    public void calculateSquare() {
+
+    }
+
+    public void calculatePerimeter() {
+
+    }
```
Comment (aperavoznikau-idt) [inline Lesson07/src/com/gmail/panser81/lesson07/part1/Figure.java:30]: Better to use abstract method

File: Lesson07/src/com/gmail/panser81/lesson07/part1/Figure.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package com.gmail.panser81.lesson07.part1;
+
+public class Figure {
+    private String name;
+    private double square;
+    private double perimeter;
```
Comment (aperavoznikau-idt) [inline Lesson07/src/com/gmail/panser81/lesson07/part1/Figure.java:6]: These are calculated parameters. Better don't use as a fields

**Review by aperavoznikau-idt — CHANGES_REQUESTED** [review]

File: Lesson07/src/com/gmail/panser81/lesson07/part1/Circle.java (hunk 1 above)
Comment (panser81) [reply Lesson07/src/com/gmail/panser81/lesson07/part1/Circle.java]: fixed

File: Lesson07/src/com/gmail/panser81/lesson07/part1/Figure.java (hunk 3 above)
Comment (panser81) [reply Lesson07/src/com/gmail/panser81/lesson07/part1/Figure.java]: updated

**Review by aperavoznikau-idt — APPROVED** [review]

**Pull request merged**